apps/site/assets/ts/stop/components/DepartureTimes.tsx

```diff
@@ -269,6 +269,7 @@ interface DepartureTimesProps {
   directionId: DirectionId;
   schedulesForDirection: ScheduleWithTimestamp[] | undefined;
   overrideDate?: Date;
+  onClick?: any;
```

Comment:
question: Is there a case where an `onClick` would not be passed? Wondering if it is safe for this to not be an optional prop

Reply:
It doesn't have to be optional. Thought these were used on the current page but that's not the case